docs/platform_operators/warning-regarding-route_crd-usage

```diff
@@ -0,0 +1,7 @@
+Currently, when a user creates a route the Cloud Controller creates a route resource as defined by [Route CRD](https://docs.google.com/document/d/1DF7eTBut1I74w_sVaQ4eeF74iQes1nG3iUv7iJ7E35U/edit#heading=h.1z6hvc8yon6k) writes it to the K8s API.
```

Comment:
Should be in markdown. Needs some formatting, ie heading seperate from text. See other docs in this directory for example.